src/modules/Content/index.tsx

```diff
@@ -99,44 +100,65 @@ export function ContentUI(props: ContentUIProps): ReactElement {
 /** @notExported */
 export interface ContentProps extends ContentSharedProps {}
 
+interface ContentState {
+    searchTerm: string;
+}
+
 /**
  * Renders Content
  */
 export function Content(props: ContentProps): ReactElement {
+    const { state, handlers } = useChangeHandlers<ContentState>({ searchTerm: '' });
+    const searchTerm: string = state.searchTerm;
+    /**
+     * Split the text and insert styled spans (yellow bg-color)
+     * when there are matches for the search term
+     *
+     * @param body - the Content HTML string
+     *
+     * @returns - a string
+     */
+    const highlightedBody = useCallback(
```

Comment:
`useCallback` is generally for functions that are going to be passed to some react thing to be called on some event like a click, timer, etc. Here is doesn't really do anything since you're calling it directly and passing the return value down. Instead you want to memoize the actual results with `useMemo()` any time either body or search term changes.

Generally I'd say we should create a new type here with `chunks` to always pass down (and if no search term it's just one big chunk) just like we do in Transcript, but in this case I think keeping it as one big string with markup applied is probably ok for now at least. If/when we add the ability to click through matches and need to scroll to each of them separately, we'll want refs and this will need to be split into react elements that wrap with `<mark>` and have refs (you'll still use innerHtml and all that, just broken up into small bits).

One other thing is, what happens if you search for something like `spa` and there's a `<span />` in the html?